docs/source/user_guide/installation/install-from-docker.rst

```diff
@@ -189,6 +189,28 @@ If you want to exit the Docker container's shell, you can simply type:
 
       exit
 
+
+Using Git Inside a Running Docker Container
+-------------------------------------------
+
+1. Setting up git configuration
```

Comment:
We should add some information here. Something like - "You might require re-configuring git while running the docker container for the first time."?

Reply:
Sure, looking into it.